pkg/controller/logstorage/secrets/secret_controller.go

```diff
@@ -559,7 +558,7 @@ func (c *elasticKeyPairCollection) trustedBundle(cm certificatemanager.Certifica
 	return cm.CreateTrustedBundle(certs...)
 }
 
-func (c *elasticKeyPairCollection) component(bundle certificatemanagement.TrustedBundle) render.Component {
+func (c *elasticKeyPairCollection) component() render.Component {
```

Comment:
WDYT about renaming this to `internalESComponent()`? I think that rename was done in the original PR https://github.com/tigera/operator/pull/3135, IIUC. I think it would help reconcile this backport with the "original" change. I realize it perhaps doesn't make total sense though since the internal  vs external isn't something in 1.32.